docs/getting_started/tutorial.md

```diff
@@ -667,7 +667,7 @@ This method is now available from your templates. Update `blog_index_page.html`
         <h2><a href="{% pageurl post %}">{{ post.title }}</a></h2>
 
         <!-- Add this: -->
-        {% with post.main_image as main_image %}
+        {% with post.specific.main_image as main_image %}
```

Comment:
I'm surprised this is required, there's already a `{% with post=post.specific %}` block enclosing this code just a few lines above 🤔

Reply:
Oh you're right, my mistake. I wasn't working with an exact copy of the code when I ran into an issue.